inference-engine/tests/ngraph_helpers/ngraph_functions/src/utils/ngraph_helpers.cpp

```diff
@@ -171,6 +168,14 @@ std::vector<ov::runtime::Tensor> interpretFunction(const std::shared_ptr<Functio
             throw std::runtime_error("Parameter: " + input.get_node_shared_ptr()->get_friendly_name() + " was not find in input parameters");
         }
         auto inputTensor = inputIt->second;
+        Shape inputShape;
+        if (input.get_partial_shape().is_dynamic()) {
+            inputShape = inputTensor.get_shape(); // For dynamic input, use tensor's shape
+        } else {
+            inputShape = input.get_shape();
```

Comment:
it should be the same for static scenario. Looks like extra code. The example of migrated tests [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/src/single_layer/softmax.cpp)